packages/test/test-end-to-end-tests/src/test/messageSize.spec.ts

```diff
@@ -154,7 +159,7 @@ describeNoCompat("Message size", (getTestObjectProvider) => {
 		assertMapValues(remoteMap, messageCount, largeString);
 	});
 
-	itExpects(
+	itExpects.only(
```

Comment:
@andre4i I need some help in debugging this. The runtime resubmits the batches separately, but some weird behavior is happening on service/socket side.

It looks like the lambda is sending all the batches back together in a single array for the "op" event. This causes the socket to disconnect, but for some reason the `DocumentDeltaConnection` is not recognizing the socket has been disconnected. Because it doesn't see the disconnect, the container hangs until the test times out.

I'm somewhat concerned about that first point if this is how the system is intended to work.

Reply:
This sounds oddly familiar with the existing chunking bug in the local server (ADO:2690)

Reply:
It was local server, and looks to be that exact ADO issue. Should we skip this test in the meantime?

Reply:
Yes